src/main/java/com/minecolonies/coremod/tileentities/TileEntityColonyBuilding.java

```diff
@@ -262,15 +309,59 @@ public NBTTagCompound writeToNBT(@NotNull final NBTTagCompound compound)
             //todo: actually do something about it and not spam the server
         }
         */
+        compound.setTag(TAG_INVENTORY, itemHandler.serializeNBT());
+        if (customName != null)
+        {
+            compound.setString(TAG_CUSTOM_NAME, customName);
+        }
         compound.setInteger(TAG_COLONY, colonyId);
         compound.setBoolean(TAG_MIRROR, mirror);
         return compound;
     }
 
-    @Override
     public boolean isUseableByPlayer(@NotNull final EntityPlayer player)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)